fail2ban/test_fail2ban.py

```diff
@@ -0,0 +1,154 @@
+# stdlib
+import subprocess  # noqa
+
+# third party
+import mock
+
+# project
+from tests.checks.common import AgentCheckTest
+
+
+# NOTE: Feel free to declare multiple test classes if needed
+
+class TestFail2ban(AgentCheckTest):
+    """Basic Test for fail2ban integration."""
+    CHECK_NAME = 'fail2ban'
+
+    def setUp(self):
+        self.config = {
+            "instances": [
+                {
+                    "sudo": True,
+                },
+            ],
+        }
+        self.load_check(self.config)
+
+    def test_execute_command(self):
+        with mock.patch("subprocess.Popen") as popen:
```

Comment:
Now that you're using `get_subprocess_output` you need to update this patch (the correct target to patch is `_fail2ban.get_subprocess_output` instead of `subprocess.Popen`)

Same thing on `test_execute_command_sudo` below